functions/account.yml

```diff
@@ -0,0 +1,18 @@
+CreateAccount:
+  handler: src/handler/account-handler.createAccount
+  events:
+    - http:
+        path: accounts
+        method: post
+GetAccounts:
+  handler: src/handler/account-handler.getAccounts
+  events:
+    - http:
+        path: accounts
+        method: get
+GetAccount:
+  handler: src/handler/account-handler.getAccount
+  events:
+    - http:
+        path: account/{id}
```

Comment:
path should be plural `accounts/{id}`

Reply:
Done, closing this pull request and creating new one for develop branch, this one is master.